src/main/java/org/jitsi/jigasi/xmpp/CallControl.java

```diff
@@ -146,23 +191,33 @@ else if (allowedJid == null)
 
                 String callResource = ctx.getCallResource();
 
-                gateway.createOutgoingCall(ctx);
+                if("transcribe:".equals(to))
```

Comment:
Can we move this to a constant somewhere, and name it something that is even less likely to use used as a sip address (e.g. jitsi_transcribe)? This will be hidden by a UI somewhere anyway.